clang-tools-extra/clang-tidy/utils/LexerUtils.h

```diff
@@ -89,10 +89,6 @@ SourceLocation findNextAnyTokenKind(SourceLocation Start,
   }
 }
 
-std::optional<Token>
-findNextTokenIncludingComments(SourceLocation Start, const SourceManager &SM,
-                               const LangOptions &LangOpts);
-
```

Comment:
I think you could keep that, and just use `Lexer::findNextToken` as its implementation (it would avoid the `/*IncludeComments=*/true` everywhere)